frontend/src/app/components/wp-new/wp-new-split-view.component.ts

```diff
@@ -26,13 +26,13 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {WorkPackageCreateController} from 'core-components/wp-new/wp-create.controller';
+import {WorkPackageCreateComponent} from 'core-components/wp-new/wp-create.component';
 import {Component} from '@angular/core';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/8bc9e1f18f1494c2afdb99f865cf858298ae755e/frontend/src/app/components/wp-new/wp-new-split-view.component.ts#L31"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>